sql/base/LegionCore_auth_2020_04_03.sql

```diff
@@ -29,7 +31,7 @@ CREATE TABLE `account` (
   `failed_logins` int(10) unsigned NOT NULL DEFAULT '0',
   `locked` tinyint(3) unsigned NOT NULL DEFAULT '0',
   `lock_country` varchar(2) NOT NULL DEFAULT '00',
-  `last_login` timestamp NOT NULL DEFAULT '0000-00-00 00:00:00',
+  `last_login` timestamp NOT NULL DEFAULT '1970-01-01 00:00:01',
```

Comment:
I do not think we need this without `NO_ZERO_IN_DATE`, `NO_ZERO_DATE`?
Probably better if I add some instructions about setting the SQL mode permanently.

Reply:
We should basically set the SQL mode permanently like how it is instructed here:
https://www.sitepoint.com/quick-tip-how-to-permanently-change-sql-mode-in-mysql/
and just keep the relevant MySQL 5.6 modes:
`ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION`

Reply:
Yeah MySQL 8 had that on by default.  Mine is shared with many other databases so I didn't want to blanket change SQL modes in case someone was doing the same,   It appears to be happy if you change that mode import the databases and set the mode back to what it was, but I'm not sure what long-term issues that may cause with MySQL 8 if they change support of that.

Reply:
I am not confident enough that nowhere else in LegionCore a zero date is used.
So if we want to "officially" support MySQL 5.7 and higher, we should mandate the "permanent" removal of those SQL modes.
If we don't it might break existing functionality which I would not want to risk with the limited amount of time I have to support this anyway.

Users like you, with specific MySQL setups, can always ignore that mandate, and solve it like you did by modifying the query.

Reply:
I'm okay with that, I didn't like changing the default value but figured it was worth me testing to see if it had any impact without making users change it.  Personally I hate that they changed it like that, doesn't look very nice and isn't as obvious that it's a default or minimum value for date.

Reply:
I'll write that up for the README sometime this week or next weekend.
So for your changes feel free to assume anyone using it will be using the 5.6 SQL modes.